openforcefield/utils/toolkits.py

```diff
@@ -4081,6 +4111,10 @@ def __init__(self):
                 f"available. {self._toolkit_installation_instructions}"
             )
 
+        out = subprocess.check_output(["antechamber", "-L"])
+        ambertools_version = out.decode("utf-8").split("\n")[1].split()[3].strip(":")
+        self._toolkit_version = ambertools_version
```

Comment:
This isn't perfect, but I do agree that it beats the traditional "we don't have any clue which version of ambertools is down there" (and its neighbor "`conda install -c conda-forge ambertools` --> You've installed AmberTools17, from pypi")

Reply:
Agreed; I've added a TODO to note that this isn't perfect and we welcome an improvement